rundeckapp/src/test/groovy/rundeck/interceptors/ApiProjectSelectInterceptorSpec.groovy

```diff
@@ -26,11 +26,21 @@ class ApiProjectSelectInterceptorSpec extends Specification implements Intercept
 
         where:
         project    | controller    | action                    | match
-        'xProject' | 'menu'        | 'apiExecutionsRunningv14' | true
         '*'        | 'menu'        | 'apiExecutionsRunningv14' | false
-        '***'      | 'menu'        | 'apiExecutionsRunningv14' | true
-        '*Project' | 'menu'        | 'apiExecutionsRunningv14' | true
```

Comment:
instead of delete, you could use match=false

Reply:
Done